File: _assets/output_gen/jsModels/ElectionState.js

//---------------------------------------------------------------------------

export default class ElectionState {
  constructor(options) {

    this.Id = null;
    this.Mode = null;
    this.ElectionStart = null;
    this.ElectionEnd = null;
    this.DateUpdated = null;
    this.UpdatedBy = null;

    if(options) {
      this.init(options);
    }
  }

  //---------------------------------------------------------------------------
  init(options) {

    this.Id = options.Id || this.Id;
    this.Mode = options.Mode || this.Mode;
    this.ElectionStart = options.ElectionStart || this.ElectionStart;
    this.ElectionEnd = options.ElectionEnd || this.ElectionEnd;
    this.DateUpdated = options.DateUpdated || this.DateUpdated;
    this.UpdatedBy = options.UpdatedBy || this.UpdatedBy;

  }  
  //---------------------------------------------------------------------------  
}

File: _assets/output_gen/Entities/ElectionState.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace OB.Data.Entities
{
    public class ElectionState
    {
        //--------------------------------------------------------
        [Key]
        public int Id { get; set;}

        [Required,StringLength(16)]
        public int Mode { get; set;}

        public DateTime ElectionStart { get; set;}

        public DateTime ElectionEnd { get; set;}

        public DateTime DateUpdated { get; set;}

        public string UpdatedBy { get; set;}

        //--------------------------------------------------------
    }
}